apps/cde-ui/src/app/components/file-upload/file-upload.component.ts

```diff
@@ -40,8 +40,23 @@ export type FileLoadError = FileTypeError | FileParseError;
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class FileUploadComponent<T, OptionsT> {
-  /** Title */
-  readonly fileTitle = input<string>();
+  /** Notification for action required */
+  readonly actionNotification = input<string>();
+
+  /** If file upload is successful */
+  readonly uploaded = input<boolean>();
```

Comment:
This input is not necessary. You already have this information from a combination of the values of `file` and `errorMessage`